src/server/gc_worker/gc_worker.rs

```diff
@@ -959,17 +950,16 @@ where
         callback: Callback<Vec<LockInfo>>,
     ) -> Result<()> {
         GC_COMMAND_COUNTER_VEC_STATIC.physical_scan_lock.inc();
-        self.check_is_busy(callback).map_or(Ok(()), |callback| {
-            self.worker_scheduler
-                .schedule(GcTask::PhysicalScanLock {
-                    ctx,
-                    max_ts,
-                    start_key,
-                    limit,
-                    callback,
-                })
-                .or_else(handle_gc_task_schedule_error)
-        })
+
+        self.worker_scheduler
+            .schedule(GcTask::PhysicalScanLock {
```

Comment:
Don't know when `phsical_scan_lock` is triggered. Would it block the outer process if it's not been executed for a long time.

Reply:
It's not expected to use any longer. It's used by Green GC, but Green GC does seem to be enabled in a short time.